features/step_definitions/your_contributions_steps.rb

```diff
@@ -9,7 +9,7 @@
 end
 
 When(/^I click edit on the first step section$/) do
-  find_link('edit').click
+  find_link('Edit').click
```

Comment:
On the second step we are using the page object to click the edit. Shall we do the same here? By the way, the step find_link "Edit" is too generic since now depending on what step you're, there could be two edit links. The page object with the right section will solve this. Wdyt?